java/Nikiforov-Oleksandr/src/com/zoolatech/lecture3/tasks/_1/Cello.java

```diff
@@ -0,0 +1,12 @@
+package com.zoolatech.lecture3.tasks._1;
+
+public class Cello extends MusicalInstrument {
+    public Cello(String sound, String name) {
+        super(sound, name);
+    }
+
+    @Override
+    public String toString() {
+        return "Cello";
```

Comment:
You can just return the instrument name; right now the instrument passed in the constructor might differ from the value returned by `toString`.